src/pages/_document.tsx

```diff
@@ -1,34 +0,0 @@
-import React from 'react'
```

Comment:
we should not delete this file

Reply:
In this case, we use "document" for defining styled-components in our application. We have to use "document" in our app, but the configuration will be distinct depending on the Design System. I did not want to add the empty "document" file as there is no Design System in this template.